Procfile

```diff
@@ -0,0 +1 @@
+web:node src/index.js
```

Comment:
you can run the production script instead

Reply:
you can use web:npm start 

but you have to configure start script well

Reply:
Fixed it , you can review again